client/core/core.go

```diff
@@ -1110,7 +1134,7 @@ func (c *Core) connectedWallet(assetID uint32) (*xcWallet, error) {
 // synching, this also starts a goroutine to monitor sync status, emitting
 // WalletStateNotes on each progress update.
 func (c *Core) connectWallet(w *xcWallet) (depositAddr string, err error) {
-	err = w.Connect(c.ctx) // ensures valid deposit address
+	err = w.Connect() // ensures valid deposit address
```

Comment:
`xcWallets` are now stopped via their `Disconnect` method, as done now in `(*Core).Run`.

Reply:
Any reason to use the `dex.Connector` for `asset.Wallet` at all now?

Reply:
I suppose just for On and Wait.  Could revisit this.